build/commands/lib/syncUtils.js

```diff
@@ -186,6 +190,14 @@ function syncChromium(program) {
     }
   }
 
+  if ((gclientWithoutRevision && (syncWithForce || chromiumNeedsUpdate)) && fs.existsSync(path.join(config.srcDir, 'chrome', 'VERSION'))) {
+    // Checking out chromium manually if necessary, as no `--revsion` flag is being passed to gclient.
+    if (util.runGit(config.srcDir, ['rev-parse', requiredChromiumRef], true) == null) {
+      util.runGit(config.srcDir, ['fetch', 'origin', requiredChromiumRef])
+    }
+    util.runGit(config.srcDir, ['checkout', '--force', requiredChromiumRef])
```

Comment:
I'm not sure if this is similar to `git reset --hard` + `git checkout`. You need to revert all changes, including those that are not touched by `git checkout --force` call.

Reply:
If that's the case, I guess `git reset --hard` alone would suffice?

Pushed a fix, but if you want me to add the `checkout` as well, let me know.